database/rrdhost.c

```diff
@@ -741,6 +756,66 @@ struct label *create_label(char *key, char *value, LABEL_SOURCE label_source)
     return result;
 }
 
+struct label *add_label_to_list(struct label *l, char *key, char *value, LABEL_SOURCE label_source)
+{
+    struct label *lab = create_label(key, value, label_source);
+    lab->next = l;
+    return lab;
+}
+
+int list_contains(struct label *head, struct label *check)
+{
+    while (head!=NULL)
```

Comment:
Please add a space between values and comparison.

Reply:
Will fix.